### ISSUE TITLE
add surface as expected value for highway=footway, path, cycleway, track, residential...

### ISSUE THREAD

**Issue by matkoniecz** [simonpoole/beautified-JOSM-preset#73]
For example on adding highway=track (unfortunately only from presets) "tracktype" tag is autoadded (without value).

It allows quicker adding data and remind about filling this info.

I think that it is reasonable to add surface tag for highway
 - path
 - footway
 - cycleway
 - track
 - pedestrian
 - residential
 - unclassified
 - service
 - tertiary
 - tertiary_link
 - secondary
 - secondary_link
 - primary
 - primary_link
 - trunk
 - trunk_link

And maybe also
 - motorway
 - motorway_link

**Comment by simonpoole** [comment]
Surface and smoothness is already available as an optional tag on a lot if not all of these,

**Comment by matkoniecz** [comment]
I know, but it would be nice for this tag to be prefilled in tag list like tracktype for tracks.

It would save typing during survey.

**Comment by simonpoole** [comment]
"prefilled" ? as in not optional or?

**Comment by matkoniecz** [comment]
To reproduce: 

- create highway=footway by using "dedicated footway" preset entry. Note that "surface" field is visible with an empty value.
- leave tagging menu and go back to the map view
- select that highway=footway again
- enter tagging menu
- Note that "surface" field with an empty value is not visible.

I see now that I should report it as more general, and in Vespucci bugtracker. I think that for values that are mandatory empty fields should be displayed also on editing existing objects.

**Comment by simonpoole** [comment]
Yes this has nothing to do with the presets itself, currently the decision in vespucci is not to apply the preset automatically when an existing st of tags is being edited. That might change one day, but is independent of the presets themselves.
